com.basiomeuspuga.Lector.json

```diff
@@ -0,0 +1,143 @@
+{
+	"app-id":"com.basiomeuspuga.Lector",
+	"runtime":"org.kde.Platform",
+	"runtime-version":"5.12",
+	"sdk":"org.kde.Sdk",
+	"command":"lector",
+	"rename-icon":"Lector",
+	"rename-desktop-file":"lector.desktop",
+	"rename-appdata-file":"lector.appdata.xml",
+	"finish-args":[
+		"--filesystem=host",
+		"--socket=x11",
+		"--socket=wayland",
+		"--device=dri",
+		"--share=ipc",
+		"--share=network"
+	],
+	"build-options":{
+		"cflags":"-O2",
+		"cxxflags":"-O2"
```

Comment:
these are defaults with this version of the runtime, best to drop them.

Reply:
Done.